packages/commons/src/logging/loggerMiddleware.ts

```diff
@@ -29,6 +29,7 @@ export const customFormat = winston.format.printf(
     const { userId, organizationId, correlationId } = getLoggerMetadata();
 
     const lines = message
+      .toString()
```

Comment:
minor fixes on the log system. Prevent calling `split` on non-string message type